src/views/network/Network.vue

```diff
@@ -435,6 +459,25 @@
                     })
                 }
             },
+            toggleSearch (key) {
+                if (this.active) {
+                    if (key.ctrlKey && key.keyCode === 70) {
+                        key.preventDefault()
+                        this.search_enabled = !this.search_enabled
+                    } else if (key.keyCode === 27) {
+                        this.search_enabled = false
+                        this.clearSearch()
+                    }
+                    if (this.search_enabled) {
+                        this.$nextTick(() => this.$refs.search.focus())
+                    } else {
+                        this.$nextTick(() => this.$refs.panel.focus())
+                    }
```

Comment:
It's more natural to focus the search field if it's visible instead of hiding it, now that we have the Esc key to do it.